unity-client/Assets/Scripts/MainScripts/DCL/Models/SceneAssets/SceneObject.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+[System.Serializable]
+public class SceneObject 
+{
+
+    [System.Serializable]
+    public class ObjectMetrics
+    {
+        public int meshes;
+        public int bodies;
+        public int materials;
+        public int textures;
+        public int triangles;
+        public int entities;
+    }
+
+
+    public string id;
+    public string asset_pack_id;
+    public string name;
+    public string model;
+    public string thumbnail;
+    public List<string> tags;
+
+    public string category;
+    public Dictionary<string, string> contents;
+
+    public string created_at;
+    public string updated_at;
+
+    public ObjectMetrics metrics;
+    public string script;
+    public bool isFavorite = false;
+
+    string baseUrl = "https://builder-api.decentraland.org/v1/storage/contents/";
```

Comment:
We should centralize all these URLs as constants under a single static class, this will make the kernel migration easier down the road and help us maintain the endpoints easier.

Reply:
Do you already have some script with all the constants or should I create one?

Reply:
I think we can use this file for this:

https://github.com/decentraland/explorer/blob/a60b375ad78273999bbe47a69ff2daca49ef0151/unity-client/Assets/Scripts/MainScripts/DCL/Configuration/Configuration.cs

Reply:
Moved to that file